code/modules/mining/ores_coins.dm

```diff
@@ -380,22 +380,60 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 
 /obj/item/coin/plasma
 	cmineral = "plasma"
+	desc = "You really shouldn't keep this in the same pocket as a lighter."
 	icon_state = "coin_plasma_heads"
 	materials = list(MAT_PLASMA = 400)
 	credits = 80
 
+/obj/item/coin/plasma/bullet_act(obj/item/projectile/P)
+	if(!QDELETED(src) && !P.nodamage && (P.damage_type == BURN))
+		log_and_set_aflame(P.firer, P)
+
+/obj/item/coin/plasma/attackby(obj/item/I, mob/living/user, params)
+	if(is_hot(I))
+		log_and_set_aflame(user, I)
+	return
+
+/obj/item/coin/plasma/proc/log_and_set_aflame(mob/user, obj/item/I)
+	var/turf/T = get_turf(src)
+	message_admins("Plasma coin ignited by [key_name_admin(user)]([ADMIN_QUE(user, "?")]) ([ADMIN_FLW(user, "FLW")]) in ([COORD(T)] - [ADMIN_JMP(T)]")
+	log_game("Plasma coin ignited by [key_name(user)] in [COORD(T)]")
+	investigate_log("was <font color='red'><b>ignited</b></font> by [key_name(user)]", "atmos")
+	user.create_log(MISC_LOG, "Plasma coin ignited using [I]", src)
+	fire_act()
+
+/obj/item/coin/plasma/fire_act(datum/gas_mixture/air, exposed_temperature, exposed_volume, global_overlay = TRUE)
+	..()
+	atmos_spawn_air(LINDA_SPAWN_HEAT | LINDA_SPAWN_TOXINS, 5) // 2 is the "correct" ammount, but its super lame. Im sure this wont have ramifications on the plasma market.
```

Comment:
What do you mean with `correct amount`? The amount that a single plasma sheet would spawn?